generators/client-2/templates/src/main/webapp/app/components/login/login.html

```diff
@@ -1,6 +1,8 @@
 <div class="modal-header">
-    <button type="button" class="close" data-dismiss="modal" aria-hidden="true"
-            ng-click="vm.cancel()">&times;</button>
+    <button aria-label="Close" data-dismiss="modal" class="close" type="button" (click)="dismiss('closed')"><span aria-hidden="true">x</span>
+    </button>
+    <!--<button type="button" class="close" data-dismiss="modal" aria-hidden="true"
```

Comment:
If i remove it the only way to close the window will be to click outside of the modal. I think the cross on the top right corner is very useful.